packages/mui-base/src/Field/Message/FieldMessage.types.tsx

```diff
@@ -0,0 +1,10 @@
+import type { BaseUIComponentProps } from '../../utils/types';
+
+export type FieldMessageOwnerState = {
+  disabled: boolean;
+  valid: boolean | null;
+};
+
+export interface FieldMessageProps extends BaseUIComponentProps<'p', FieldMessageOwnerState> {
+  show?: keyof ValidityState;
```

Comment:
Have you considered accepting an array of values here (so that the message is shown if any of the specified errors are present)?

Reply:
Not sure if that makes sense?